src/locale/fr-FR.ts

```diff
@@ -0,0 +1,89 @@
+import type { StaticTextDefaultValue } from '../../index';
+
+/**
+ * @author Adrien Foulon
+ * @github https://github.com/tofandel
+ * Author is native French
+ */
+const FR_FR: StaticTextDefaultValue = {
+  toolbarTips: {
+    bold: 'Gras',
+    underline: 'Souligné',
+    italic: 'Italique',
+    strikeThrough: 'Barré',
+    title: 'Titre',
+    sub: 'Indice',
+    sup: 'Exposant',
+    quote: 'Citation',
+    unorderedList: 'Liste',
+    orderedList: 'Liste numérique',
+    task: 'Liste de tâche',
+    codeRow: 'Code intégré',
+    code: 'Bloc de code',
+    link: 'Lien',
+    image: 'Image',
+    table: 'Table',
+    mermaid: 'Organigramme',
+    katex: 'Formule',
+    revoke: 'Annuler',
+    next: 'Refaire',
+    save: 'Sauvegarder',
+    prettier: 'Sublimer',
+    pageFullscreen: 'Passer en pleine page',
+    fullscreen: 'Passer en plein écran',
+    preview: 'Prévisualiser',
+    htmlPreview: 'Prévisualiser le HTML',
+    catalog: 'Table des matières',
+    github: 'Code source de l\'editeur',
+  },
+  titleItem: {
+    h1: 'Titre niveau 1',
+    h2: 'Titre niveau 2',
+    h3: 'Titre niveau 3',
+    h4: 'Titre niveau 4',
+    h5: 'Titre niveau 5',
+    h6: 'Titre niveau 6',
+  },
+  imgTitleItem: {
+    link: 'Ajouter depuis une url',
+    upload: 'Télécharger une image',
+    clip2upload: 'Télécharger et recadrer',
+  },
+  linkModalTips: {
+    title: 'Ajouter un lien',
```

Comment:
There is a problem with how the title of the link modal is handled because it's used by both the `link` and `img` components a shortcut was taken to use one title with something appended
https://github.com/imzbf/md-editor-v3/blob/main/MdEditor/layouts/Modals/Link.tsx#L49

![image](https://user-images.githubusercontent.com/6115458/202578453-58e61be4-e024-44f3-945a-4eee6ce6ccd4.png)

This doesn't allow for correct translations and needs to be changed